ocaml-lsp-server/test/e2e/__tests__/__snapshots__/workspace-symbol.test.ts.snap

```diff
@@ -0,0 +1,1485 @@
+          "line": 52,
+        },
+      },
+      "uri": "file://<testWorkspacePath>/lib/lib.ml",
+    },
+    "name": "compare_length_with",
+  },
+  Object {
+    "kind": 12,
+    "location": Object {
+      "range": Object {
+        "end": Object {
+          "character": 35,
+          "line": 59,
+        },
+        "start": Object {
+          "character": 0,
+          "line": 59,
+        },
+      },
+      "uri": "file://<testWorkspacePath>/lib/lib.ml",
+    },
+    "name": "cons",
+  },
+  Object {
+    "kind": 12,
+    "location": Object {
+      "range": Object {
+        "end": Object {
+          "character": 22,
+          "line": 64,
+        },
+        "start": Object {
+          "character": 0,
+          "line": 64,
+        },
+      },
+      "uri": "file://<testWorkspacePath>/lib/lib.ml",
+    },
+    "name": "hd",
+  },
+  Object {
+    "kind": 12,
+    "location": Object {
+      "range": Object {
+        "end": Object {
+          "character": 27,
+          "line": 69,
+        },
+        "start": Object {
+          "character": 0,
+          "line": 69,
+        },
+      },
+      "uri": "file://<testWorkspacePath>/lib/lib.ml",
+    },
+    "name": "tl",
+  },
+  Object {
+    "kind": 12,
+    "location": Object {
+      "range": Object {
+        "end": Object {
+          "character": 30,
+          "line": 74,
+        },
+        "start": Object {
+          "character": 0,
+          "line": 74,
+        },
+      },
+      "uri": "file://<testWorkspacePath>/lib/lib.ml",
+    },
+    "name": "nth",
+  },
+  Object {
+    "kind": 12,
+    "location": Object {
+      "range": Object {
+        "end": Object {
+          "character": 41,
+          "line": 81,
+        },
+        "start": Object {
+          "character": 0,
+          "line": 81,
+        },
+      },
+      "uri": "file://<testWorkspacePath>/lib/lib.ml",
+    },
+    "name": "nth_opt",
+  },
+  Object {
+    "kind": 12,
+    "location": Object {
+      "range": Object {
+        "end": Object {
+          "character": 28,
+          "line": 89,
+        },
+        "start": Object {
+          "character": 0,
+          "line": 89,
+        },
+      },
+      "uri": "file://<testWorkspacePath>/lib/lib.ml",
+    },
+    "name": "rev",
+  },
+  Object {
+    "kind": 12,
+    "location": Object {
+      "range": Object {
+        "end": Object {
+          "character": 40,
+          "line": 92,
+        },
+        "start": Object {
+          "character": 0,
+          "line": 92,
+        },
+      },
+      "uri": "file://<testWorkspacePath>/lib/lib.ml",
+    },
+    "name": "init",
+  },
+  Object {
+    "kind": 12,
+    "location": Object {
+      "range": Object {
+        "end": Object {
+          "character": 42,
+          "line": 98,
+        },
+        "start": Object {
+          "character": 0,
+          "line": 98,
+        },
+      },
+      "uri": "file://<testWorkspacePath>/lib/lib.ml",
+    },
+    "name": "append",
```

Comment:
That's what gets produced due to the `include List`. The `cmt.cmt_sourcefile` point to the file where we use the `include` (`lib/lib.ml`) but the location is the location where the function (`append` in this case) is defined (in the `List` module somewhere e.g in the `_opam` folder). Therefore, we end up at a wrong place in the file where we use the `include` when we click on the symbol. Would that be ok to ignore `symbols` from `include` for now?

Reply:
Should have been addressed in https://github.com/ocaml/ocaml-lsp/pull/398/commits/8b31c489f2a152674479f3c2a768b2edd7d9d967